index.html

```diff
@@ -11,12 +11,13 @@ <h3>Multi-Screen Window Placement Demo (<a href="https://github.com/michaelwasse
 <h4 id="warning" hidden=true style="color: red;"></h4>
 <div class="controls">
   <div id="openWindowControls" style="display:flex; align-items:center;">
-    <button id="openWindowButton">Open window</button>
-    URL: <input id="openWindowUrlInput" style="flex-grow:1;" value="./popup.html"></input>
+    <button id="openWindowButton" style="flex-shrink: 0;">Open window</button>
+    URL: <input id="openWindowUrlInput" style="flex-grow: 1;min-width: 81px;max-width: 125px;" value="./popup.html"></input>
     left: <input id="openWindowLeftInput" style="width:50px;" size=5 value="100" type="number"></input>
     top: <input id="openWindowTopInput" style="width:50px;" size=5 value="50" type="number"></input>
     width: <input id="openWindowWidthInput" style="width:50px;" size=5 value="830" type="number"></input>
     height: <input id="openWindowHeightInput" style="width:50px;" size=5 value="750" type="number"></input>
+    <label for="fullscreenPopupInput"> fullscreen: </label> <input type="checkbox" id="fullscreenPopupInput" name="fullscreenPopup" value="true"><br>
```

Comment:
optional nit: `<label>` is nice! use that for other inputs here and in popup.html too (or I can easily follow up afterwards)

Reply:
Done.